lib/packer/builders/amazon.rb

```diff
@@ -12,7 +12,7 @@ def initialize
           'instance_type',
           'region',
           'source_ami',
-          'ssh_username'
+          ['ssh_username', 'communicator']
```

Comment:
Ah yeah I think I might have screwed it up but my intention was to make the setting exclusive. In my particular use case, I don't want to supply an ssh_username, just the winrm communicator and winrm specific information. If I didn't do this, I would fail the validation.

Thanks!